packages/blade/src/components/Chip/__tests__/__snapshots__/ChipGroup.native.test.tsx.snap

```diff
@@ -0,0 +1,10513 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Oh man, 10k plus snapshots. 

TBH i'm not sure at this point this will be much of use, can we reduce few snapshots? I think it's getting this big because we are snapshotting [every possible](https://github.com/razorpay/blade/pull/1547/files#diff-2ca24370c91077981af15bcac075041c67c52dd367133334f1e6d23e1255731bR8-R10) variant. 

I would say it's better to do visual testing for these than adding this many snapshots

Reply:
I have removed snaps for all sizes and variants.